ocs_ci/utility/deployment_openshift_logging.py

```diff
@@ -0,0 +1,377 @@
+"""
+This module deploys the openshift-logging on the cluster
+EFK stack
+"""
+
+import logging
+import json
+
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.ocs.resources.pod import get_all_pods, get_pod_obj
+from ocs_ci.utility import templating
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.utility.retry import retry
+from tests import helpers
+
+logger = logging.getLogger(__name__)
+
+
+def create_namespace(yaml_file):
+    """
+    Creation of namespace "openshift-operators-redhat"
+    for Elasticsearch-operator and "openshift-logging"
+    for ClusterLogging-operator
+
+    Args:
+        yaml_file (str): Path to yaml file to create namespace
+
+    Example:
+        create_namespace(yaml_file=constants.EO_NAMESPACE_YAML)
+
+    """
+
+    namespaces = ocp.OCP(kind=constants.NAMESPACES)
+
+    logger.info("Creating Namespace.........")
+    assert namespaces.create(yaml_file=yaml_file), 'Failed to create namespace'
+    logger.info("Successfully created Namespace")
+
+
+def create_elasticsearch_operator_group(yaml_file, resource_name):
+    """
+    Creation of operator-group for Elastic-search operator
+
+    Args:
+        yaml_file (str): Path to yaml file to create operator group for
+            elastic-search
+        resource_name (str): Name of the operator group to create for
+            elastic-search
+
+    Returns:
+        Bool: True if operator group for elastic search is created
```

Comment:
This should be `bool`, not `Bool`. The point here is that it is the actual type in python that is returned. We can fix them in another PR as I don't want to hold this up any longer, however these need to be the actual type exactly how they are in python.